lib/api/model/model.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:json_annotation/json_annotation.dart';
 
+import '../route/messages.dart';
```

Comment:
Hmm, I don't like having lib/api/model/ import from lib/api/route/. If it's needed for the model, that feels like a sign it's part of the model and not part of how particular routes interact with the model.

I think these can just stay in this file.

Reply:
(When this file gets to feel unwieldy, I think the thing to do is to split pieces out by subject area: like a `lib/ap/model/message.dart`, akin to the `narrow.dart` and `submessage.dart` and so on that we already have.)